File: Famoser.MassPass.Data/Entities/Communications/Request/ContentEntityRequest.cs
using System;
using Famoser.MassPass.Data.Entities.Communications.Request.Base;

namespace Famoser.MassPass.Data.Entities.Communications.Request
{
    public class ContentEntityRequest : ApiRequest
    {
        public Guid ContentId { get; set; }
        public string VersionId { get; set; }
    }
}

File: Famoser.MassPass.Api/src/Models/Request/ContentEntityRequest.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 22/05/2016
 * Time: 23:01
 */

namespace Famoser\MassPass\Models\Request;


use Famoser\MassPass\Models\Request\Base\ApiRequest;

class ContentEntityRequest extends ApiRequest
{
    public $ContentId;
    public $VersionId;
}